ppa/archive/models.py

```diff
@@ -584,6 +584,11 @@ def has_fulltext(self):
         HathiTrust, Gale, or EEBO-TCP)."""
         return self.source in [self.HATHI, self.GALE, self.EEBO]
 
+    @property
+    def work_type(self):
+        """Work type formatted for COinS metadata (matches Solr field format)."""
+        return self.get_item_type_display().lower().replace(" ", "-")
```

Comment:
Good solution. Please update the `index_data` method to use this property (~ line 1009) so we can guarantee they stay in sync.